lib/ControlSystemsBase/src/pid_design.jl

```diff
@@ -536,36 +534,94 @@ function convert_pidparams_to_standard(param_p, param_i, param_d, form::Symbol)
     end
 end
 
+"""
+    Kp, Ti, Td = convert_pidparams_to_parallel(param_p, param_i, param_d, form)
+
+Convert parameters from form `form` to `:parallel` form.
+
+The `form` can be chosen as one of the following
+* `:standard` - ``K_p(1 + 1/(T_i s) + T_d s)``
+* `:series` - ``K_c(1 + 1/(τ_i s))(τ_d s + 1)``
+* `:parallel` - ``K_p + K_i/s + K_d s``
+"""
+function convert_pidparams_to_parallel(param_p, param_i, param_d, form::Symbol)
+    if form === :parallel
+        return @. (param_p, param_i, param_d)
+    elseif form === :series
+        # param_i = 0 would result in division by zero, but typically indicates that the user wants no integral action
+        param_i == 0 && return @. param_p * (1, 0, param_d)
+        return @. (param_p *
+            ((param_i + param_d) / param_i, 1 / param_i, param_d))
+    elseif form === :standard
+        param_i == 0 && return @. param_p * (1, 0, param_d)
+        return @. param_p * (1, 1 / param_i, param_d)
+    else
+        throw(ArgumentError("form $(form) not supported."))
+    end
+end
+
 """
     param_p, param_i, param_d = convert_pidparams_from_standard(Kp, Ti, Td, form)
 
 Convert parameters to form `form` from `:standard` form. 
 
 The `form` can be chosen as one of the following
-* `:standard` - ``K_p(1 + 1/(T_i s) + T_ds)``
+* `:standard` - ``K_p(1 + 1/(T_i s) + T_d s)``
 * `:series` - ``K_c(1 + 1/(τ_i s))(τ_d s + 1)``
 * `:parallel` - ``K_p + K_i/s + K_d s``
 """
 function convert_pidparams_from_standard(Kp, Ti, Td, form::Symbol)
     if form === :standard
-        return Kp, Ti, Td
+        return @. (Kp, Ti, Td)
```

Comment:
```suggestion
        return Kp, Ti, Td
```
this `@.` didn't do anything